packages/renderer/src/modules/store/reducers/workspace/slice.ts

```diff
@@ -30,6 +30,10 @@ export function createWorkspaceSlice() {
         .addCase(getWorkspace.rejected, (state, action) => {
           state.status = 'failed';
           state.error = action.error.message || 'Failed to get workspace';
+        })
+        .addCase(createProject.fulfilled, state => {
+          state.status = 'reload';
+          state.error = null;
```

Comment:
I think we can just return the new project from the `createProject` method on preload, and just push the new project on the current state. Doing a full reload implies scanning the current working directory for decentraland scenes, and reading each scene's `scene.json` file.